ironfish-cli/package.json

```diff
@@ -77,6 +77,8 @@
     "inquirer": "8.2.5",
     "json-colorizer": "2.2.2",
     "keccak": "3.0.4",
+    "natural-orderby": "3.0.2",
```

Comment:
Neither of these are "new" - they were both used by oclif's table. They have been promoted to direct dependencies